### ISSUE TITLE
Bug Report - Tab shows icon

### ISSUE THREAD

**Issue by TayYuanGeng** [microsoft/terminal#1862]
# Environment

```
Microsoft Windows [Version 10.0.18362.207]
Windows Terminal version (if applicable): 0.2.1715.0
```

# Steps to reproduce

Start Windows Terminal,
Spam open different types of terminal (using hotkeys),
Preferably spamming Ubuntu, cmd and PowerShell,
Some tabs collapsed into showing just icon while others are expanded.

# Expected behavior

1. The tabs should be standardised and collapsed only when there are too many tabs to fit in.
2. The tabs will only expand when there are enough space to fit in.

# Actual behavior

Screenshot belows show what it looks like:
Some of the tabs are expanded while others are not.
![image](https://user-images.githubusercontent.com/50003738/60782508-8706df00-a179-11e9-97e6-2135f168e77b.png)
Cleared remaining tabs and only left with collapsed tab, it does not expand every tab but the last tab.
![image](https://user-images.githubusercontent.com/50003738/60782652-27f59a00-a17a-11e9-9d0d-7d97e99a3df1.png)

**Comment by zadjii-msft** [comment]
@TayYuanGeng So we're definitely not doing any auto-collapsing of the tabs to icons. What's probably happening here is the shell isn't starting up entirely, and it's not setting the title. 

After playing with this I can get this to repro myself. Focusing the tab also doesn't seem to set the title anymore. However, if you have the tab that's broken focused, then unfocus the window and re-focus the window, the title will update.

We must be missing some title change event, but for whatever reason we think that we _did_ actually set the title. Either that, or there might be some bug in the tab control itself, since the window focus thing seems _insane_ to me.

**Comment by TayYuanGeng** [comment]
Apparently, I tried to focus and unfocus. It DOES update BUT for ONLY the last tab. In between the expanded tabs are those collapsed tab.

**Comment by DHowett-MSFT** [comment]
This'll be fixed by #1205. Thanks for the report!
